src/components/table/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact'
+import Analysis from '../../models/analysis'
+
+const Table = ({ data }: { data: Analysis[] }) => (
+  <table>
+    <th />
```

Comment:
Can you make this match https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#More_Examples

Reply:
Added